package.json

```diff
@@ -25,7 +25,7 @@
     "@internetarchive/feature-feedback": "^0.1.2",
     "@internetarchive/field-parsers": "^0.1.3",
     "@internetarchive/histogram-date-range": "^0.1.6",
-    "@internetarchive/infinite-scroller": "^0.1.2",
+    "@internetarchive/infinite-scroller": "0.1.3",
```

Comment:
Suggest "^0.1.3"

About using the caret ^: https://nodesource.com/blog/semver-tilde-and-caret/

Reply:
@nsharma123 is there a specific reason why you removed the caret here?

i also just learned this versioning, more info on source [here](https://javascript.plainenglish.io/the-math-behind-npm-versioning-5236f8c821c2)

![image](https://user-images.githubusercontent.com/1281581/173426763-7f43c3cc-2ca9-42c9-b24e-24939274dddd.png)